miasm2/expression/expression.py

```diff
@@ -399,7 +418,7 @@ def visit(self, cb, tv=None):
         return self
 
     def copy(self):
-        return ExprId(self.name, self._size)
+        return ExprId(self.name, self.size)
```

Comment:
In my opinion, you could use a private attribute (`_size`) inside this method and avoid an indirection. It does not break the encapsulation.